code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -2205,3 +2205,18 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	glass_icon_state = "pina_colada"
 	glass_name = "Pina Colada"
 	glass_desc = "If you like pina coladas, and getting caught in the rain... well, you'll like this drink."
+
+
+/datum/reagent/consumable/ethanol/pruno // pruno mix is in drink_reagents
+	name = "pruno"
+	color = "#E78108"
+	description = "Fermented prison wine made from fruit, sugar, and despair. Security loves to confiscate this, which is the only kind thing Security has ever done."
+	boozepwr = 85
+	taste_description = "your tastebuds being individually shanked"
+	glass_icon_state = "glass_orange"
+	glass_name = "glass of pruno"
+	glass_desc = "Fermented prison wine made from fruit, sugar, and despair. Security loves to confiscate this, which is the only kind thing Security has ever done."
+
+/datum/reagent/consumable/ethanol/pruno/on_mob_life(mob/living/carbon/M)
+	SEND_SIGNAL(M, COMSIG_ADD_MOOD_EVENT, "disgust", /datum/mood_event/verygross)
```

Comment:
dont do this, add disgust to the mob

Reply:
isn't that what I'm doing

Reply:
(this isn't meant to be rude, it's a genuine question)

Reply:
there's a seperate system for disgust similar to how things like drugginess work. that automatically assigns the moods etc and makes you puke after a while

Reply:
use adjust_disgust() and an if prob to randomly give spikes of DISGUSTANG
```suggestion
	if(prob(30)) // dont wanna spam disgust
		M.adjust_disgust(15)
```

Reply:
the prob doesn't make sense. just lower the amount from 15 to 5 and you have the same effect without the RNG